frontend/sozisel-app/src/components/SessionResultScreen/SessionResultEvents/EventResultDetails/Modules/QuizResultDetails/QuizResultDetails.scss

```diff
@@ -0,0 +1,89 @@
+@import "../../../../../../styles/styles";
+
+
+.QuizResultDetails {
+    height: calc(100% - #{$navbar-height} - #{6 * $spacing-09} - #{$spacing-08});
```

Comment:
Do you really need that weird calculations? I can understand the navbar height but not the other spacinigs